test/unit/request_test.rb

```diff
@@ -66,4 +66,16 @@ def test_location_after_safe_location
     assert_equal 'RD', req.safe_location.country_code
     assert_equal 'US', req.location.country_code
   end
-end
+  def test_geocoder_remove_port_from_addresses
```

Comment:
- [ ] These tests look good. I think what you have is sufficient. The only thing I have to add is potentially adding tests that verify your code handles bad input.